pom.xml

```diff
@@ -130,6 +131,7 @@
         <module>mirror-maker-agent</module>
         <module>tracing-agent</module>
         <module>test</module>
+        <module>strimzi-container</module>
```

Comment:
Should we name it `testcontainer`? The `strimzi` seems unnecessary to me and only `continer` is not enough to differentiate it from the regular containers.

Reply:
Sure thing